src/components/DpUploadFiles/DpUploadFiles.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/vue'
-import DpUploadFiles from './'
+import { DpUploadFiles } from './index'
```

Comment:
why that change? I mean, it works, but why do You think this change is an improvement?

Reply:
Ah, the './index' part is not needed, or what are you referring to?

Reply:
```
import DpUploadFiles from './'
```
should still work or not?

Reply:
Ok, I think the option to import like that gets lost if we import two components in the index file (I believe it's not related to removing `export * from './DpUploadFiles.vue'`). But the truth is that we never import like that
`import DpUploadFiles from './'`